pages/delete_Accept_Button.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable react/button-has-type */
+const deleteButton: React.FunctionComponent = () => (
+  // eslint-disable-next-line react/jsx-no-comment-textnodes
+  <div className="flex space-x-4">
+    <div>
+      <button className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">
```

Comment:
Suggestion: reuse the Button component in components/Button. Great work with the flex container. I think this is small enough to be placed directly in the page source!